src/containers/guest-home-page/signup-form/SignupForm.js

```diff
@@ -8,11 +9,16 @@ import { style } from './signup-form.style'
 
 const SignupForm = ({ handleSubmit, handleChange, handleBlur, data, errors }) => {
   const { t } = useTranslation()
+  const [buttonDisabled, setButtonDisabled] = useState(true)
   const { inputVisibility: passwordVisibility, showInputText: showPassword } = useInputVisibility(errors.password)
   const { inputVisibility: confirmPasswordVisibility, showInputText: showConfirmPassword } = useInputVisibility(
     errors.confirmPassword
   )
 
+  const handleButtonChange = () => {
```

Comment:
What button? I think it should be named "onAgreementChange".

Reply:
done.